module/VuFind/tests/unit-tests/src/VuFindTest/Navigation/AccountMenuTest.php

```diff
@@ -243,4 +185,44 @@ protected function getOldDefaultMenuConfig(): array
             ],
         ];
     }
+
+    /**
+     * Data provider for testRequiredConfiguration
+     *
+     * @return \Iterator<string, array>
+     */
+    public static function requiredConfigurationProvider(): \Iterator
+    {
+        yield 'Missing group settings' => [
+            ['Account' => []],
+        ];
+        yield 'Missing menu item settings' => [
+            [
+                'Account' => [
+                    'label' => 'Test',
+                    'MenuItems' => [[]],
+                ],
+            ],
+        ];
+    }
+
+    /**
+     * Test required configuration.
+     *
+     * @param array  $config                      Account menu configuration
+     * @param string $expectedExceptionClass      Expected exception class
+     * @param string $expectedExceptionMsgMatches Expected exception message regexp
+     *
+     * @return void
+     */
+    #[\PHPUnit\Framework\Attributes\DataProvider('requiredConfigurationProvider')]
+    public function testRequiredConfiguration(
+        array $config,
+        string $expectedExceptionClass = BadConfig::class,
+        string $expectedExceptionMsgMatches = '/^Missing required setting: /'
+    ) {
```

Comment:
Might as well include type:

```suggestion
    ): void {
```